transformation/src/jade-schema/tables/assay_activity.table.json

```diff
@@ -0,0 +1,139 @@
+{
+  "name": "assay_activity",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_started",
```

Comment:
Does ENCODE actually expose metadata for start/end time? If not, we won't be able to populate these columns.

Reply:
Yes